ledger-tool/src/main.rs

```diff
@@ -847,6 +848,16 @@ fn main() {
                 .hidden(hidden_unless_forced())
                 .help(BlockVerificationMethod::cli_message()),
         )
+        .arg(
+            Arg::with_name("unified_scheduler_handler_threads")
+                .long("unified-scheduler-handler-threads")
+                .value_name("THREADS")
+                .takes_value(true)
+                .validator(|s| is_within_range(s, 1..))
+                .global(true)
+                .hidden(hidden_unless_forced())
+                .help(DefaultSchedulerPool::cli_message()),
+        )
```

Comment:
the actual builder method invocation order and lack of `.global(true)` or not are slightly different from the `solana-validator` counterpart. however, this is my best attempt to be consistent within respective surrounding convention (ie. they're already inconsistent)...